.github/workflows/release.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
         uses: actions/checkout@v2
       - name: "Unshallow (required by goreleaser)"
         run: git fetch --prune --unshallow
+      - name: Get the version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
```

Comment:
I don't see is consuming this anywhere; why do we set it?

Reply:
It used at the end of the file to build the manifest

Reply:
Then I think you need to do `::set-env` and not `::set-output`